### ISSUE TITLE
Installation instructions should be a bit longer

### ISSUE THREAD

**Issue by SirVer** [coolwanglu/neovim.as#5]
I never did anything with web technology, Atom shell.... whatnot. Could you expand the three lines in the README to a bit more substantial help to get everything installed?

**Comment by rygwdn** [comment]
Would be nice to use a setup similar to https://github.com/atom/atom-shell-starter to make the bootstrapping process easier.

**Comment by jmlucjav** [comment]
indeed, specially if one is on Windows (not sure if it is supported yet?)

**Comment by aquillano** [comment]
I'm curious how you connect this GUI to NeoVim running on a remote server.

**Comment by rockneverdies55** [comment]
:+1:

**Comment by raghur** [comment]
This worked for me on Windows 7
1. install elecron 
   
   ```
   npm install electron-prebuilt -g
   ```
2. Download latest neovim binary for windows from appveyor. Keep it somewhere on the path.
3. Clone neovim repo and move runtime folder to "C:\Program Files\Neovim\share\nvim"
4. Then clone this repo and install
   
   ```
   git clone <this repo>
   cd <folder>
   apm install . && grunt
   ```
5. Launch with `Electron .`

I also had to copy `~/.vim` to `~/.nvim` and `~/.vimrc` to `~/.nvimrc`

Required a little bit of head scratching but I agree that having this on the front page will help

**Comment by trusktr** [comment]
A `.app` for OS X will be awesome. :D

**Comment by simonweil** [comment]
How do you run `apm`?
Running on MAC, I have installed electron with brew: `brew cask install electron` but no `apm` command exists...
Do I need to install Atom?

**Comment by trusktr** [comment]
@simonweil https://github.com/atom/apm

**Comment by simonweil** [comment]
Thanks!

**Comment by cdvv7788** [comment]
I agree with this. I have no idea on how to make this run.
